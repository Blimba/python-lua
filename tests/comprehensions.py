
a = [i * j for i in range(5) for j in range(3) if i * j % 2 == 0 and i > 0 and j > 0]

for item in a:
    print(item)

lst = ["a","b","c","d","e"]
b = {lst[i]: i ** 2 for i in range(5)}

for k in lst:
    print(k,b[k])