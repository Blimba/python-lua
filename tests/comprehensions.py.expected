2
2
4
6
4
8
a	0.0
b	1.0
c	4.0
d	9.0
e	16.0
